### PR TITLE
Update Notification interface to match correct API

### PR DESCRIPTION
**Pull request by ScreamZ**
Improve document payload according to real documentation

https://docs.kuzzle.io/core/2/api/payloads/notifications/#document-notification

Can you ping me on release @rolljee  @etrousset ? Thanks :)

### CODE REVIEW

File: src/types/Notification.ts [hunk 1]
```diff
@@ -62,7 +62,21 @@ export interface DocumentNotification<
...
+     */
+    _source: TDocContent;
+    /**
+     * Document unique ID.
+     * `null` if the notification is from a real-time message.
+     */
+    _id: string | null;
+    /**
+     * List of fields that have been updated (only available on document partial updates).
+     */
```
Comment (ScreamZ) [inline src/types/Notification.ts:77]: Not sure what « only available on document partial updates » means. Is it missing? Is it `null`?

Comment (ScreamZ) [reply src/types/Notification.ts:77]: I just tested, and the field is missing, so the type is good.

**Review by etrousset — APPROVED** [review]

**Pull request merged**